source/common/quic/envoy_quic_utils.h

```diff
@@ -125,5 +126,13 @@ bssl::UniquePtr<X509> parseDERCertificate(const std::string& der_bytes, std::str
 // not supported, return 0 with error_details populated correspondingly.
 int deduceSignatureAlgorithmFromPublicKey(const EVP_PKEY* public_key, std::string* error_details);
 
+// Return a connection socket which can get information from io_handle, but not
+// be able to modify it.
```

Comment:
Make io_handle const, if it can't be modified?

Reply:
io_handle can't be const. The socket can still call readv/writev on io_handle which modifies io_handle, but can't set socket options and close the socket. 

I updated the comment to be more accurate.